### DISCUSSION TITLE
Does stopping the plotter also stop the copying process?

### DISCUSSION THREAD

**Discussion by cyperbg** [madMAx43v3r/chia-plotter#553] [Q&A]
When I want to stop the plotter I always wait for the last ready plot to finish copying which is usually when the next plot is in Phase 2.
Is there a way to stop the plotter, but keep the copying process alive?

**Comment by ESP4Ever** [comment]
Hi,

Just press CTRL-C once, the SW will tell you that it will abort but finish the current plot and its copying phase.

**Reply by cyperbg** [reply]
> Hi,
> 
> Just press CTRL-C once, the SW will tell you that it will abort but finish the current plot and its copying phase.

So you mean there is no way to stop the plotting without stopping the copying?

**Reply by ESP4Ever** [reply]
you can abort everything by closing the terminal and to copy the plot manually
